adidt/cli/main.py

```diff
@@ -441,3 +441,21 @@ def jif(ctx, node_type, reboot, filename):
         d.update_current_dt(reboot=reboot)
     else:
         raise Exception("Other node types not implemented")
+
+
+@cli.command()
+@click.option(
+    "--profile",
+    "-p",
+    default=None,
+    help="",
+    type=click.Path(exists=True),
+)
+@click.pass_context
+def profile2dt(ctx, profile):
+    """Generate devicetree from Profile Configuration Wizard files
+    """
+    adrv = adidt.adrv9009_pcbz()
+    adrv.parse_profile(profile)
+    adrv.gen_dt()
+    print(f'Wrote {adrv.output_filename}')
```

Comment:
This needs to be more generic. Eventually we'll need to support other profiles. Can you change this to use something like ast.literal_eval or eval to take like the part or board name?

Reply:
Hi @tfcollins, apologies for the delay I was away for a few days.

> This needs to be more generic. Eventually we'll need to support other profiles.

Understood, I'll make the change.

I was also working on a parser for `talise_config.c` that I'll add to this MR. I think it can help fill more fields in the devicetree template.